indra/assemblers/pybel/assembler.py

```diff
@@ -414,20 +415,27 @@ def belgraph_to_signed_graph(
 
 
 def _combine_edge_data(relation, subj_edge, obj_edge, stmt_hash, evidences):
-    edge_data = {pc.RELATION: relation}
+    edge_data = {
+        pc.RELATION: relation,
+        pc.ANNOTATIONS: {},
+    }
     if subj_edge:
         edge_data[pc.SUBJECT] = subj_edge
     if obj_edge:
         edge_data[pc.OBJECT] = obj_edge
     if stmt_hash:
-        edge_data['stmt_hash'] = stmt_hash
+        edge_data[pc.ANNOTATIONS]['stmt_hash'] = stmt_hash
```

Comment:
@bgyori this is where we should put more statement-level metadata from INDRA

Reply:
Yes, the UUID could be useful but it would take a bit of refactoring to expose it here since we only propagate the stmt_hash down to function.

Reply:
Refactored to propagate the statement instance in dee6f1f0